engine/api/workflow/repository.go

```diff
@@ -99,21 +101,16 @@ func extractWorkflow(ctx context.Context, db *gorp.DbMap, store cache.Store, p *
 		return nil, allMsgs, err
 	}
 
-	mods := []workflowtemplate.TemplateRequestModifierFunc{
```

Comment:
Mods should be define here next to CheckAndExecute, maybe you should refact it to get projKeys instead of proj.

Reply:
done